### ISSUE TITLE
download limit

### ISSUE THREAD

**Issue by ghost** [mmshivesh/WebImageScraper#1]
is there a line where the download limit is set, I need a limit of 1000

**Comment by mmshivesh** [comment]
Yup! I'll upgrade this soon! Hang tight!

**Comment by ghost** [comment]
thanks, I am using alternative method for now to get 1k images

**Comment by mmshivesh** [comment]
Yup, there are implementations out there with other packages which enable unto 1000 images
